x/records/keeper/grpc_query_deposit_record_test.go

```diff
@@ -12,11 +12,21 @@ import (
 
 	keepertest "github.com/Stride-Labs/stride/testutil/keeper"
 	"github.com/Stride-Labs/stride/testutil/nullify"
+	"github.com/Stride-Labs/stride/x/records/keeper"
 	"github.com/Stride-Labs/stride/x/records/types"
 )
 
+func createNDepositRecord(keeper *keeper.Keeper, ctx sdk.Context, n int) []types.DepositRecord {
```

Comment:
Looks like this was added to get the tests running, we must have deleted it at some point for no good reason?